packages/tokens/src/global/fonts.ts

```diff
@@ -38,9 +38,9 @@ export const fontWeights: FontWeightTokens = {
 export const fontFamilies: FontFamilyTokens = {
   fontFamilyBase:
     // eslint-disable-next-line @fluentui/max-len
-    "'Segoe UI', 'Segoe UI Web (West European)', -apple-system, BlinkMacSystemFont, Roboto, 'Helvetica Neue', sans-serif",
```

Comment:
Might as well remove BlinkMacSystemFont too, as it's the equivalent of -apple-system in Chrome browser (on MacOS and iOS)